Formula/open-sp.rb

```diff
@@ -12,14 +12,35 @@ class OpenSp < Formula
     sha256 "d7b79be390f3c2b2a823e1156d896200db397dffb6cb6e6712d27539e05ca18b" => :yosemite
   end
 
+  depends_on "docbook" => :build
+  depends_on "xmlto" => :build
+  depends_on "ghostscript" => :build
+
   def install
     system "./configure", "--disable-debug", "--disable-dependency-tracking",
                           "--prefix=#{prefix}",
                           "--mandir=#{man}",
-                          "--disable-doc-build",
                           "--enable-http",
-                          "--enable-default-catalog=#{etc}/sgml/catalog",
-                          "--enable-default-search-path=#{HOMEBREW_PREFIX}/share/sgml"
+                          "--enable-default-catalog=#{etc}/sgml/catalog"
+
+    ENV["XML_CATALOG_FILES"] = "#{etc}/xml/catalog"
     system "make", "pkgdatadir=#{share}/sgml/opensp", "install"
   end
+
+  test do
+    sgml_path = testpath.join("eg.sgml")
+    sgml_path.write <<-SGML.undent
```

Comment:
just use `EOS.undent` as that's pretty standard throughout the formulae.

Reply:
I was getting rubocop audit complaints about meaningful Heredoc names